README.md

```diff
@@ -84,10 +84,11 @@ POST to [http://localhost:3000/api/admin/registerTemp](http://localhost:3000/ap
 
 Now you can login to admin-frontend with given credentials.
 
-## Note about production
+### Note about production
 
-When deploying application to production, endpoint should initially be open(remove the AuthGuards), and after creation of admin, it should be closed asap. 
+When deploying application to production, endpoint should initially be open, and after creation of admin, it should be closed ASAP. The endpoint is toggled by environment variable `SUPER_ADMIN_FEATURES`. Set its value to "yes" to allow registering admins via the endpoint and unset the variable (or set as "no") to disable the endpoint afterwards.
```

Comment:
Should this be boolean, and not yes or no

Reply:
It was intentionally yes and not "true", that way it's not accidentally on. Also, environment variables are strings anyway, so we'd have to use a separate string comparison === "true" function to convert it to boolean. Doable, but still wish to do so?